packages/translations/src/languages/tr.ts

```diff
@@ -295,6 +295,7 @@ export const trTranslations: DefaultTranslationsObject = {
     titleDeleted: '{{label}} {{title}} başarıyla silindi.',
     true: 'Doğru',
     unauthorized: 'Yetkisiz',
+    unsavedChanges: undefined,
```

Comment:
Did you get any errors when you ran the translation script? I see a few are `undefined`.